charts/atlantis/templates/statefulset.yaml

```diff
@@ -138,6 +138,12 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if or .Values.gitconfig .Values.gitconfigSecretName }}
+          lifecycle:
+            postStart:
+              exec:
+                command: ["/bin/sh", "-c", "cp /etc/secret-gitconfig/gitconfig /home/atlantis/.gitconfig && chown atlantis /home/atlantis/.gitconfig"]
```

Comment:
Doesn't the `chown` command require i.e. `user:group` e.g. `atlantis:atlantis` ?

Reply:
I don't believe it's required to provide the group for the chown command to work. I believe in this case the file permissions would end up being `atlantis:root`. Overall I took what was done [previously](https://github.com/runatlantis/helm-charts/pull/203/files#diff-a3b675e5633e3d9602003c23ede42b91c32bc10f4aed39235b204cff10c11bbdL135) but happy to add the group if thats preferred.